src/services/hubspot/baseRequest.ts

```diff
@@ -94,7 +73,15 @@ const baseRequest: Request = async (client, {
         });
     }
 
-    return await res.json();
+    let result;
+
+    try {
+      result = await res.json();
+    } catch (e) {
+      return {};
+    }
```

Comment:
Do we need a console.warn here? It seems strange to expect a JSON and just return a blank object.

Reply:
added error message
`console.warn("Failed to parse response as JSON. Returning empty result")`